From 2e485e16881f0aad726dd401df6e9f2eb96f64dd Mon Sep 17 00:00:00 2001
From: Sugizaki Yukimasa <i.can.speak.c.and.basic@gmail.com>
Date: Thu, 4 Jan 2018 23:58:06 +0900
Subject: [PATCH 148/454] vcsm: Define cache operation constants in user header

Without this change, users have to use raw values (1, 2, 3) to specify
cache operation.

Signed-off-by: Sugizaki Yukimasa <i.can.speak.c.and.basic@gmail.com>
---
 drivers/char/broadcom/vc_sm/vc_sm_knl.h | 5 -----
 include/linux/broadcom/vmcs_sm_ioctl.h  | 5 +++++
 2 files changed, 5 insertions(+), 5 deletions(-)

--- a/drivers/char/broadcom/vc_sm/vc_sm_knl.h
+++ b/drivers/char/broadcom/vc_sm/vc_sm_knl.h
@@ -27,11 +27,6 @@ enum vc_sm_lock_cache_mode {
 	VC_SM_LOCK_NON_CACHED,
 };
 
-/* Cache functions */
-#define VCSM_CACHE_OP_INV       0x01
-#define VCSM_CACHE_OP_CLEAN     0x02
-#define VCSM_CACHE_OP_FLUSH     0x03
-
 /* Allocate a shared memory handle and block. */
 int vc_sm_alloc(struct vc_sm_alloc_t *alloc, int *handle);
 
--- a/include/linux/broadcom/vmcs_sm_ioctl.h
+++ b/include/linux/broadcom/vmcs_sm_ioctl.h
@@ -79,6 +79,11 @@ enum vmcs_sm_cache_e {
 	VMCS_SM_CACHE_BOTH,
 };
 
+/* Cache functions */
+#define VCSM_CACHE_OP_INV       0x01
+#define VCSM_CACHE_OP_CLEAN     0x02
+#define VCSM_CACHE_OP_FLUSH     0x03
+
 /* IOCTL Data structures */
 struct vmcs_sm_ioctl_alloc {
 	/* user -> kernel */
